### PR TITLE
Show in which primary storage volumes are allocated in instance details page and volume migration pop-ups

### PR DESCRIPTION
**Pull request by winterhazel**
### Description

This PR changes the volume tab in the instance details page, the volume migration pop-up and the instance migration with volumes pop-up to show admins in which primary storage the volumes are allocated.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

### Screenshots (if appropriate):
<details>
<summary>
Volumes tab in the instance details page
</summary>

Screenshot from 2024-03-26 09-43-58

</details>

<details>
<summary>
Volume migration pop-up
</summary>

Screenshot from 2024-03-26 09-44-51

</details>

<details>
<summary>
Instance migration with volumes pop-up
</summary>

Screenshot from 2024-03-26 09-45-26

</details>

### How Has This Been Tested?

1. In the root admin account, I accessed an instance's page and verified that the volumes tab showed the `Storage` column.
2. In an account that did not have permission to view the storage a volume is allocated, I accessed an instance's page and verified that the volumes tab did not show the `Storage` column.
3. In the root admin account, I opened the volume migration pop-up and verified that it showed the volume's current storage;
4. In the root admin account, I opened the instance migration with volumes pop-up and verified that it showed the current storage volumes were allocated in.

### CODE REVIEW

File: ui/public/locales/en.json [hunk 1]
```diff
@@ -585,6 +585,7 @@
 "label.currency": "Currency",
 "label.current": "Current",
 "label.currentpassword": "Current password",
+"label.current.storage": "Current storage",
```
Comment (GaOrtiga) [inline ui/public/locales/en.json:588]: @winterhazel Shouldn't this come before `label.currentpassword`?

**Comment by sureshanaparti** [comment]
@blueorangutan ui

**Review by bernardodemarco — APPROVED** [review]
LGTM. I manually tested in the QA environment.

Volumes tab in VM details page:
<details>
    <summary>Before</summary>

image

</details>

<details>
    <summary>After</summary>

image
</details>

Volume migration modal:
<details>
    <summary>Before</summary>

image
</details>

<details>
    <summary>After</summary>

image
</details>

VM migration to another primary storage modal:
<details>
    <summary>Before</summary>

image
</details>

<details>
    <summary>After</summary>

image
</details>

**Review by BryanMLima — APPROVED** [review]
CLGTM

**Pull request merged**